### PR TITLE
add governance and contributors

### PR DESCRIPTION
**Pull request by rootfs**

### CODE REVIEW

**Comment by rootfs** [comment]
@marceloamaral @sunya-ch @williamcaban @SamYuan1990  @wangchen615 @jichenjc @Feelas @huoqifeng @KaiyiLiu1234 @husky-parul @kenplusplus @sallyom @ruomengh 

Please review and provide any missing info and feedback ASAP.

Thanks!

File: Contributors.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+| Ji Chen | jichenjc | Testing | IBM | China |
+| Sam Yang | SamYuan1990 | CI | IBM | China |
+| William Caban Babilonia | williamcaban | Kepler Deployment | Red Hat | USA |
+| Kayi Liu | KaiyiLiu1234 | Kepler Model Server | Red Hat | Cananda |
+| Parul Singh | husky-parul  | Kepler Documentation | Red Hat | USA |
+# Reviewers
+| Name | GH ID | Expertise | Affiliation | Location |
+|---|---|---|---|---|
+| Huo Qi Feng | huoqifeng | Kepler Deployment | IBM | China |
+| Filip Skirtun | Feelas | Kepler Deployment | Intel | |
```
Comment (Feelas) [inline Contributors.md:16]: Please add Poland as location

File: Contributors.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+| Name | GH ID | Expertise | Affiliation | Location |
+|---|---|---|---|---|
+| Huamin Chen | rootfs | Kepler Core | Red Hat | USA |
+| Marcelo Carneiro do Amaral | marceloamaral | Kepler Core | IBM | Japan |
+| Sunyanan Choochotkaew (Pang) | sunya-ch | Kepler Core | IBM | Japn |
+| Ji Chen | jichenjc | Testing | IBM | China |
+| Sam Yang | SamYuan1990 | CI | IBM | China |
+| William Caban Babilonia | williamcaban | Kepler Deployment | Red Hat | USA |
+| Kayi Liu | KaiyiLiu1234 | Kepler Model Server | Red Hat | Cananda |
+| Parul Singh | husky-parul  | Kepler Documentation | Red Hat | USA |
```
Comment (husky-parul) [inline Contributors.md:11]: | Parul Singh | husky-parul | Kepler Deployment & Documentation | Red Hat | USA |

File: Contributors.md (hunk 1 above)
Comment (rootfs) [reply Contributors.md]: thanks, done

File: Contributors.md (hunk 2 above)
Comment (rootfs) [reply Contributors.md]: done

**Review by husky-parul — APPROVED** [review]
/LGTM

**Comment by rootfs** [comment]
I propose we re-evaluate the project status and committee team every 3 months.

File: Contributors.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+| Marcelo Carneiro do Amaral | marceloamaral | Kepler Core | IBM | Japan |
+| Sunyanan Choochotkaew (Pang) | sunya-ch | Kepler Core | IBM | Japn |
+| Ji Chen | jichenjc | Testing | IBM | China |
+| Sam Yang | SamYuan1990 | CI | IBM | China |
+| William Caban Babilonia | williamcaban | Kepler Deployment | Red Hat | USA |
+| Kayi Liu | KaiyiLiu1234 | Kepler Model Server | Red Hat | Cananda |
+| Parul Singh | husky-parul  | Kepler Deployment and Documentation | Red Hat | USA |
+| Chen Wang | wangchen615 | Kepler Core | IBM | USA |
+
+# Reviewers
```
Comment (marceloamaral) [inline Contributors.md:14]: @rootfs shouldn't we also be in the reviewers list?

Comment (rootfs) [reply Contributors.md:14]: some clarity the roles is needed, hopefully this request can help https://github.com/sustainable-computing-io/kepler/issues/249

Comment (marceloamaral) [reply Contributors.md:14]: @rootfs right and I think we also need to configure the owners

Kubernetes has a guideline for this [https://go.k8s.io/owners](https://go.k8s.io/owners)
```
approvers: a list of GitHub usernames or aliases that can /approve a PR
reviewers: a list of GitHub usernames or aliases that are good candidates to /lgtm a PR
```

Kubernetes has an example of this [here](https://github.com/kubernetes/kubernetes/blob/master/OWNERS)

Comment (rootfs) [reply Contributors.md:14]: having that CI would be great :D

**Pull request merged**